microsoft-support.md

```diff
@@ -0,0 +1,37 @@
+# Microsoft support for grpc-dotnet
+
+[Microsoft supports grpc-dotnet](https://LINK-TODO) on multiple operating systems, per the [Microsoft Modern Lifecycle](https://support.microsoft.com/help/30881/modern-lifecycle-policy).
+
+Support is provided for the following grpc-dotnet packages:
+* Grpc.AspNetCore
+* Grpc.AspNetCore.Server
+* Grpc.AspNetCore.Web
+* Grpc.AspNetCore.Server.ClientFactory
+* Grpc.AspNetCore.Server.Reflection
+* Grpc.Net.Client
+* Grpc.Net.ClientFactory
+* Grpc.Net.Common
+* dotnet-grpc
+
+Notes:
+* Applications must be using a [currently supported .NET release](https://dotnet.microsoft.com/platform/support/policy).
+* Minimum supported grpc-dotnet version is v2.37.0.
+* Minimum supported version is the earliest major and minor release required to obtain assisted support. Please utilize public community channels for assistance or log issues directly on GitHub for releases prior to the minimum supported version.
+* Assisted support is only available for the official builds released from https://github.com/grpc/grpc-dotnet, and no assisted support option is available for individual forks.
+* Please note that new features and security\bug fixes are provided in the latest released version and are not backported to previous versions. To obtain the latest updates and features, please upgrade to the latest available version.
```

Comment:
gRPC org policy is to provide security fixes for older releases two. For bug fixes, gRPC org support the two latest releases.

Reply:
Is it important that the two policies match here?